doc/source/tools/postprocessing/postprocessing_paraview.rst

```diff
@@ -0,0 +1,51 @@
+========================================
+Massive Data Visualization with Paraview
+========================================
+
+Lethe is developed with special care for scalability; it is intended to be used for solving extremely large problems by using massively parallel computing. As such, the output files that are obtained can sometimes be so heavy that the post-processing requires special considerations. This section covers key points to take into account in such cases.
+
+
+------------------------------
+Client-server visualization
+------------------------------
+
+The first limitation that is encountered when FEM problems contain :math:`\sim 15` M cells (when using a 32GB-RAM machine) is lack of memory. Even though the VTU files weigh 2GB, they are decompressed by Paraview when opened.
+
+
+Clusters such as those of the Digital Research Alliance of Canada allow to launch interactive jobs in which a Paraview-server can be used. Detailed instructions on how to setup and use such a server are available on this `page <https://docs.alliancecan.ca/wiki/ParaView>`_. However, here is an overview of the required steps to use Client-server visualization:

```

Comment:
```suggestion
Clusters such as those of the Digital Research Alliance of Canada allow to launch interactive jobs in which a ParaView-server can be used. Detailed instructions on how to set up and use such a server are available on this `page <https://docs.alliancecan.ca/wiki/ParaView>`_. However, here is an overview of the required steps to use Client-server visualization:

```

Reply:
These instructions are very specific to the clusters of the alliance. I think you can be more specific in this sense from the get go.